node_test/reverseproxy_test.go

```diff
@@ -22,17 +22,39 @@ import (
 )
 
 const (
-	smallResponse              = "Hello"
-	largeResponse              = "HelloWorldHelloWorld"
+	smallResponse = "Hello"
+
 	parseErrorResponseBody     = "could not parse voucher"
 	signatureErrorResponseBody = "error processing voucher"
 	proxyAddress               = ":5511"
 	bobRPCUrl                  = ":4107/api/v1"
 	destPort                   = 6622
 	otherParam                 = "otherParam"
 	otherParamValue            = "2"
+	testFileContent            = "This a simple test file used in the reverse payment proxy"
+	testFileName               = "test_file.txt"
 )
 
+func setupTestFile(t *testing.T) func() {
+	// Open the file for writing (create or truncate)
+	file, err := os.Create(testFileName)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer file.Close()
+
+	_, err = file.WriteString(testFileContent)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Should the file get removed in this case?

Reply:
https://github.com/statechannels/go-nitro/pull/1493/commits/9b0343da18afb0e467c1107251b201f86a57a187